yarn-project/circuit-types/src/interfaces/configs.ts

```diff
@@ -6,13 +6,13 @@ export type AllowedFunction =
       /** The contract address this selector is valid for */
       address: AztecAddress;
       /** The function selector */
-      selector: FunctionSelector;
+      selector?: FunctionSelector;
```

Comment:
Might we want to be so verbose as to have 
- AllowedInstance
- AllowedInstanceFunction
- AllowedClass
- AllowedClassFunction

Reply:
It is to allow the case where a full contract is added to the whitelist. For example, the auth contract in its entirety can be allowed instead of needing to add each function individually.

Reply:
Yes, so I think possible confusion comes because the type is allowed "Function". Perhaps we instead just want to rename the type to `AllowedEndpoint`